src/packages/@ncigdc/components/Aggregations/RangeFacet.js

```diff
@@ -261,91 +337,99 @@ const RangeFacet = (props: TProps) => {
   };
   const { maxDisplayed, minDisplayed } = props.state;
   return (
-    <Container style={{ ...props.style }} className="test-range-facet">
+    <Container className="test-range-facet" style={{ ...props.style }}>
       {!props.collapsed &&
         props.convertDays && (
           <Row style={{ marginBottom: '0.5rem' }}>
             <form name={`${dotField}-radio`}>
               <label
                 htmlFor={`${dotField}-years-radio`}
                 style={{ paddingRight: '10px' }}
-              >
+                >
                 <input
-                  type="radio"
-                  value="years"
-                  onChange={props.handleUnitChanged}
                   checked={props.state.selectedUnit === 'years'}
                   id={`${dotField}-years-radio`}
+                  onChange={props.handleUnitChanged}
                   style={{ marginRight: '5px' }}
-                />
+                  type="radio"
+                  value="years"
+                  />
                 Years
               </label>
               <label
                 htmlFor={`${dotField}-days-radio`}
                 style={{ paddingRight: '10px' }}
-              >
+                >
                 <input
-                  type="radio"
-                  value="days"
-                  onChange={props.handleUnitChanged}
                   checked={props.state.selectedUnit === 'days'}
                   id={`${dotField}-days-radio`}
+                  onChange={props.handleUnitChanged}
                   style={{ marginRight: '5px' }}
-                />
+                  type="radio"
+                  value="days"
+                  />
                 Days
               </label>
             </form>
           </Row>
-        )}
+      )}
       {!props.collapsed && (
         <Column>
           <Row>
             <InputLabel
+              htmlFor={`from-${dotField}`}
               style={{
                 borderRight: 0,
                 borderRadius: '4px 0 0 4px',
               }}
-              htmlFor={`from-${dotField}`}
-            >
+              >
               From:
             </InputLabel>
             <Input
-              value={props.state.fromDisplayed || ''}
-              onChange={props.handleFromChanged}
               id={`from-${dotField}`}
               key={`from-${dotField}`}
-              type="number"
-              placeholder={`eg. ${minDisplayed}`}
               max={maxDisplayed}
               min={minDisplayed}
-              title="todo"
-            />
+              onChange={props.handleFromChanged}
+              placeholder={`eg. ${minDisplayed}`}
+              style={{
+                paddingRight: '5px',
+                paddingLeft: '5px',
+              }}
+              title={`eg. ${minDisplayed}`}
```

Comment:
Talked Rosi & Brandon, can we change the title text to `Min value: eg. ${minDisplayed}`